test/test-capablities.js

```diff
@@ -0,0 +1,30 @@
+var capabilities = require('../lib/capabilities');
+var async = require('async');
+var tap = require('tap');
+
+tap.test('targetctl capabilities', function(t) {
+  var caps = ['tracing', 'metrics', 'watchdog',
+      'cpuprofile', 'heapsnapshot', 'patch'];
+  var list = capabilities.list();
+  t.ok(list, 'Capabilities are defined.');
+  if (!list) {
+    t.end();
+  } else {
+    async.mapSeries(caps,
+      function(value, cb){
+        t.ok(list.indexOf(value) >= 0,
+            'Capabilities include a check for ' + value);
+        t.doesNotThrow(function(){
+          capabilities.query(value, function(result, reason){
+            console.log('Result of', value, 'check is', result,
```

Comment:
Should be a comment in the output so that it doesn't result in malformed TAP output:
```js
t.comment('Result of %j check is %j because errors are %j', value, result, reason);
```